fury/deprecator.py

```diff
@@ -19,7 +19,7 @@
 # packaging.version.parse is a third-party utility but is used by setuptools
 # (so probably already installed) and is conformant to the current PEP 440.
 # But just if it is not the case, we use distutils
-packaging, have_pkg, _ = optional_package("setuptools.extern.packaging")
+packaging, have_pkg, _ = optional_package("packaging")
```

Comment:
Packaging is a mandatory package so you do not need optional_package function and you can import it directly. 

Thank you for the future update!

Reply:
Hi @skoudoro 

Thank you for your feedback. I'll take it into account in my next update.